crates/lsp/src/lsp.rs

```diff
@@ -799,6 +802,54 @@ impl LanguageServer {
                         hierarchical_document_symbol_support: Some(true),
                         ..DocumentSymbolClientCapabilities::default()
                     }),
+                    semantic_tokens: Some(SemanticTokensClientCapabilities {
+                        augments_syntax_tokens: Some(true),
+                        overlapping_token_support: Some(true),
+                        server_cancel_support: Some(true),
+                        requests: SemanticTokensClientCapabilitiesRequests {
+                            range: Some(true),
+                            full: Some(SemanticTokensFullOptions::Delta { delta: Some(true) }),
```

Comment:
We declare `delta` support, but I see no `pveriousResultId` management, and no `/delta` endpoints used around the code.

Am I missing something, or we do not support `delta`?

Would be very important to support this, it seems, as it impacts the way Zed should store and rewrite the semantic token data.